torch/_dynamo/variables/tensor.py

```diff
@@ -849,137 +849,93 @@ class TensorWithTFOverrideVariable(VariableTracker):
     def create(
         tx,
         tensor_variable,
-        orig_tensor_variable_source,
         torch_function_fn,
         subclass_type,
         **kwargs,
     ):
         var = TensorWithTFOverrideVariable(
             tensor_variable,
-            orig_tensor_variable_source,
             torch_function_fn,
             subclass_type,
             **kwargs,
         )
         # stash the subclass type to rewrap an output tensor if needed
-        tx.output.install_global(var.global_class_name(), subclass_type)
+        if var.global_class_name() not in tx.output.global_scope:
+            tx.output.install_global(var.global_class_name(), subclass_type)
+
         return var
 
     def __init__(
         self,
         tensor_variable,
-        orig_tensor_variable_source,
-        subclass_torch_function__func,
+        torch_function_fn,
         subclass_type,
         **kwargs,
     ):
         super().__init__(**kwargs)
         self.tensor_variable = tensor_variable
-        self.orig_tensor_variable_source = orig_tensor_variable_source
-        self.subclass_torch_function__func = subclass_torch_function__func
+        self.torch_function_fn = torch_function_fn
         self.subclass_type = subclass_type
 
-    def call_method(
-        self,
-        tx,
-        name,
-        args: "List[VariableTracker]",
-        kwargs: "Dict[str, VariableTracker]",
-    ) -> "VariableTracker":
-        # This code block implements inlining the __torch_function__ override
-        # of `call_method`.
-        from . import GetAttrVariable
+    def as_proxy(self):
+        return self.tensor_variable.as_proxy()
```

Comment:
FYI: this is going to get removed as well, the next PR will setup inheritance properly